File: app/src/main/java/com/antyzero/smoksmog/fabric/StationShowEvent.java
package com.antyzero.smoksmog.fabric;

import com.crashlytics.android.answers.ContentViewEvent;

import pl.malopolska.smoksmog.model.Station;

public class StationShowEvent extends ContentViewEvent {

    public StationShowEvent(Station station) {
        putContentId(String.valueOf(station.getId()));
        putContentName(station.getName());
        putContentType("Station");
    }
}

File: app/src/main/kotlin/com/antyzero/smoksmog/fabric/StationShowEvent.kt
package com.antyzero.smoksmog.fabric

import com.crashlytics.android.answers.ContentViewEvent

import pl.malopolska.smoksmog.model.Station

class StationShowEvent(station: Station) : ContentViewEvent() {

    init {
        putContentId(station.id.toString())
        putContentName(station.name)
        putContentType("Station")
    }
}
